keyboards/trkeyboards/trk1/readme.md

```diff
@@ -0,0 +1,27 @@
+# Tvrd Rad Keyboards TRK1
+
+![TRK1](https://i.imgur.com/nTHrBMr.jpg)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [Tvrd Rad Keyboards](https://www.instagram.com/trkeyboards/)
+* Hardware Supported: Pro Micro and clones
+* Hardware Availability: https://www.instagram.com/trkeyboards/
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```